godel_surface_detection/src/services/surface_blending_service.cpp

```diff
@@ -99,9 +113,6 @@ bool SurfaceBlendingService::init()
   visualize_process_path_client_ =
       nh.serviceClient<godel_process_path_generation::VisualizeBlendingPlan>(VISUALIZE_BLENDING_PATH_SERVICE);
 
-  // trajectory_planner_client_ = nh.serviceClient<godel_msgs::TrajectoryPlanning>(TRAJECTORY_PLANNING_SERVICE);
-
-  // trajectory_execution_client_ = nh.serviceClient<godel_msgs::TrajectoryExecution>(PATH_EXECUTION_SERVICE);
   // Read process execution service names
   std::string blend_process_service_name, scan_process_service_name;
   ph.param<std::string>("blend_process_service_name", blend_process_service_name, "blend_process_execution");
```

Comment:
I think we should not read topic or service names through parameters as we have already changed this approach in order packages and used harcoded global constants.